src/StateFile.cxx

```diff
@@ -118,11 +124,16 @@ try {
 
 	const char *line;
 	while ((line = file.ReadLine()) != nullptr) {
-		success = read_sw_volume_state(line, partition.outputs) ||
-			audio_output_state_read(line, partition.outputs) ||
-			playlist_state_restore(line, file, song_loader,
+		success =            read_sw_volume_state(line, partition.outputs);
+		success = success || audio_output_state_read(line, partition.outputs);
+		success = success || playlist_state_restore(line, file, song_loader,
```

Comment:
Why did you refactor that, and hid it in this commit?

Reply:
I refactored it to make the calls to the different save functions consistent. If you prefer I can keep the original calls.

Reply:
I see no added consistency here.